### PR TITLE
Improvements for Hackathon workshop

### PR DESCRIPTION
**Pull request by andrewwoods**
I made some improvements for the 1-hour workshop on Saturday. Some are readability, some are trying to improve the content to fit within the hour.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -177,55 +167,64 @@
         </section>
 
         <section>
-          <h3>Installation and Setup</h3>
+          <h3>Open The Terminal</h3>
+          <h4>On Mac</h4>
+          <p>Type CMD+SPACE<br>In the box type 'Terminal' and hit enter</p>
+          <h4>On Windows</h4>
+          <p>??? I don't know.</p>
```
Comment (marcysutton) [inline index.html]: We could recommend Git for Windows / Git Bash here?  https://git-for-windows.github.io/

**Review by marcysutton — CHANGES_REQUESTED** [review]
This looks great! I made one comment about the Windows section, but otherwise I think we're good to go.

File: index.html (hunk 1 above)
Comment (andrewwoods) [reply index.html]: This page on git-scm.com https://git-scm.com/download/win is a Windows EXE file, but I'm not sure what it comes with. The Git Bash looks pretty cool from what I can see on the web page. I'll add that.

**Comment by marcysutton** [comment]
Looks good!

**Pull request merged**